addons/placement/functions/fnc_boundingBoxAdjustment.sqf

```diff
@@ -0,0 +1,55 @@
+params ["_obj"];
+private ["_bb","_bbx","_bby","_bbz","_arr","_y","_z"];
+_bb = {
+	_bbx = [_this select 0 select 0, _this select 1 select 0];
+	_bby = [_this select 0 select 1, _this select 1 select 1];
+	_bbz = [_this select 0 select 2, _this select 1 select 2];
+	_arr = [];
+	0 = {
+		_y = _x;
+		0 = {
+			_z = _x;
+			0 = {
+				0 = _arr pushBack (_obj modelToWorld [_x,_y,_z]);
+			} count _bbx;
+		} count _bbz;
+		reverse _bbz;
+	} count _bby;
+	_arr pushBack (_arr select 0);
+	_arr pushBack (_arr select 1);
+	_arr
+};
+bbox = boundingBox _obj call _bb;
+bboxr = boundingBoxReal _obj call _bb;
```

Comment:
Why are these even globals? And [they should be prefixed](https://ace3mod.com/wiki/development/coding-guidelines.html#111-global-variable-naming)

Reply:
They are not globals, this file is being removed anyway before this PR is marked ready. It was being used for development

Reply:
Alright, then nevermind.